### ISSUE TITLE
Missing some key pdf features

### ISSUE THREAD

**Issue by lastlink** [albo1337/flutter_full_pdf_viewer#6]
* search document
* get, set page
* view outline, bookmarks

performance really good now, but these are key features to a pdf viewer

**Comment by jonathanloh02** [comment]
Hyperlinks such as those on content pages do not work as well

**Comment by l-k22** [comment]
I would love to see these options too

**Comment by l-k22** [comment]
> Hyperlinks such as those on content pages do not work as well

I must be taking crazy pills, I thought this was working. Has a recent Flutter update broken it or was it never working??

**Comment by albo1337** [comment]
Hey guys im sorry to answer that lately had a surgery and I'm still not in the best mood. I will work on it when im good again. Sorry...

**Comment by l-k22** [comment]
Sorry to hear that. Get well soon

**Comment by l-k22** [comment]
> Hyperlinks such as those on content pages do not work as well

I just downgraded to 1.0.1 and it's working, not sure what's changed in the new version

**Comment by albo1337** [comment]
Hey @l-k22 is this related to https://github.com/albo1337/flutter_full_pdf_viewer/issues/12 ?

**Comment by l-k22** [comment]
> Hey @l-k22 is this related to #12 ?

Yes it is, but now neither imports seem to work, not sure if this an Android X compatibility issue

**Comment by lastlink** [comment]
@l-k22 I don't know if hyperlinks every worked. Apparently you have to add support for them.

e.g. ios
```objective-c
- (BOOL)_webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)request navigationType:(UIWebViewNavigationType)navigationType {
        if (navigationType == UIWebViewNavigationTypeLinkClicked ) {
            UIApplication *application = [UIApplication sharedApplication];
            [application openURL:[request URL] options:@{} completionHandler:nil];
            return NO;
        }

        return YES;
}
```

I couldn't quite get this to work though.
